### DISCUSSION TITLE
Findings Review Comments

### DISCUSSION THREAD

**Discussion by valrdclr** [Syslifters/sysreptor#170] [General]
Hi, I was curious if there is any review feature for findings. So basically you can have typos/mistakes in your findings of course, and if the pentest lead or anybody else has to review them, then it has to be done separately (perhaps by downloading the pdf as a draft and leave comments there). Adding something to markdown is also not a good choice as it can mess it up or cause leftovers.

**Comment by MWedl** [comment]
Hi, you could prefix comments in markdown with "TODO". TODOs are highlighted in red in the markdown editor so you can quickly locate them in long texts. Additonally, TODOs generate warning messages. This helps in detecting leftovers in the report. Warning messages are shown on the publish page before generating the final PDF. 

![image](https://github.com/Syslifters/sysreptor/assets/29510012/fc76a91f-a6d4-4ec8-ae07-1ac94539a4a1)
![image](https://github.com/Syslifters/sysreptor/assets/29510012/f1926841-afa3-4697-a2be-5b8b16fa0fba)


Adding a fully fledged commenting system is technically quite challenging. Commenting fields as a whole is quite easy, but attaching comments to words of text parts (like in MS Word, Google Docs, etc.) is complex. We need to track changes and update comments positions. However, without being able to attach comments to text parts, commenting markdown is not convenient. Especially for long texts.

**Comment by MWedl** [comment]
Implemented in https://github.com/Syslifters/sysreptor/releases/tag/2024.55 (Pro only)
